Demo/MapFilter/DemoSearchLocationDataSource.swift

```diff
@@ -17,15 +17,11 @@ class DemoSearchLocationDataSource: NSObject, SearchLocationDataSource {
     private var localSearch: MKLocalSearch?
 
     private lazy var localSearchCompleter: NSObject? = {
```

Comment:
You can replace `NSObject?` with `MKLocalSearchCompleter` I guess ;)

Reply:
Yes we can 😄